exporter/otlphttpexporter/config.go

```diff
@@ -26,6 +34,9 @@ type Config struct {
 
 	// The URL to send logs to. If omitted the Endpoint + "/v1/logs" will be used.
 	LogsEndpoint string `mapstructure:"logs_endpoint"`
+
+	// The encoding to export telemetry (default: "proto")
+	Encoding EncodingType `mapstructure:"encoding"`
```

Comment:
I think we should have some validation, either as part of `Validate` or `UnmarshalText`, that enforces only `proto` or `json`.